src/translations/sqlb_es_ES.ts

```diff
@@ -2333,7 +2333,7 @@ Do you want to insert it anyway?</source>
     </message>
     <message>
         <location filename="../MainWindow.ui" line="1891"/>
-        <source>Copy the CREATE statement of the item to the clipboard</source>
+        <source>Copy the CREATE statement of the ítem to the clipboard</source>
```

Comment:
Ooops. :cry: Fixed.

Now think that are ok to merge as is.

Although for some couldn't find the dialogs where are actually being used, these are just fixed typos. All of them hopefully.

But in the GUI I'm still seeing [those same untranslated strings](https://github.com/sqlitebrowser/sqlitebrowser/pull/723#issuecomment-242979348) that seemed to be translated in the .ts like  that in line="14" but the others are not even defined there.

![portapapeles01](https://cloud.githubusercontent.com/assets/14217781/21751035/9a164efa-d5c0-11e6-8323-77f651e2d45e.png)
![portapapeles02](https://cloud.githubusercontent.com/assets/14217781/21751036/9c137f02-d5c0-11e6-8a7a-c7683dba6387.png)